examples/models/llama/export_llama_lib.py

```diff
@@ -816,6 +818,13 @@ def _export_llama(args) -> LLMEdgeManager:  # noqa: C901
 
         builder = builder.to_executorch(passes=additional_passes)
 
+    if args.verbose:
+        graph_module = builder.edge_manager.exported_program().graph_module
```

Comment:
The `EdgeManager` is modified inplace so after `to_executorch` it inserts `memory.alloc` nodes for out variant ops which are not generated in the final PTE file but used for memory allocation for out-variant ops, so the delegation count given here will be inaccurate. I think we should print this right after `to_backend` above.

Reply:
That makes sense. Let me update it. Thanks @tarun292 !